crates/krun/src/guest/bin/krun-guest.rs

```diff
@@ -15,7 +15,8 @@ use krun::utils::env::find_in_path;
 use log::debug;
 use rustix::process::{getrlimit, setrlimit, Resource};
 
-fn main() -> Result<()> {
+#[tokio::main]
```

Comment:
Some of our safety assumptions would no longer hold, e.g.

https://github.com/slp/krun/blob/44417f6e802a934f9ca164ace647f99da009546d/crates/krun/src/guest/user.rs#L21-L23

Reply:
Using tokio is a requirement of the rtnetlink crate. I think we can avoid the safety issues with something like this:
```
    let rt = tokio::runtime::Runtime::new().unwrap();
    rt.block_on(async {
        if let Err(err) = configure_network().await {
            eprintln!("Failed to configure network, continuing without it: {err}");
        }
    });
```

Reply:
Could we use tokio's current-thread scheduler?

Reply:
If we aren't planning on using tokio anywhere else, I think I feel more confident isolating it's use to `configure_network`.